### ISSUE TITLE
UI to enable / disable installed add-ons per workspace/window

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#1448]
I'm thinking plugins should not automatically apply to all windows everywhere. They should be configurable per workspace/window.  I.e. we should add one more text option for them: "Enable plugins", which would give a multi-selectable list of plugins.

In addition, we still need some plugins to be configured globally, not per-workspace/per-window basis (like RefParser). Thus, there should be "Enable plugins globally" option in app preferences.

**Comment by agrogers** [comment]
I don't really know what add ons you guys have in mind. But I wonder whether they should automatically apply to all Windows and workspaces with an option to make them apply to a selection.

Or perhaps when a new add on is installed it applies to all areas and the user has to deselect it. Or perhaps they get a choice at install.

Just don't want a user to install an add on and then not have it work because there is another setting somewhere that they need to enable.

**Comment by tuomas2** [comment]
Good point. But if plugin is manipulating how UI will look like, I think it makes sense user might want to enable something in some workspace/window but not in some else? Similarly as per-window color adjustments etc?

**Comment by tuomas2** [comment]
For now (4.0) it's all global (and hopefully not too much abused). 

Remember that when doing any tutorial videos, styling plugins should be uninstalled :-D

**Comment by agrogers** [comment]
Yes, I think it is a good feature. Just need to help the user enable it
where they want it. Could be confusing if some add ons work everywhere
always and others don't.

**Comment by tuomas2** [comment]
Perhaps it could be other way around: everything is enabled by default and there's per-workspace/per-window option to disable some of them. Similar logic as with hiding specific bookmark labels.

**Comment by agrogers** [comment]
I like it. Although I always prefer options that are positive (tick and it
is in/true) rather than tick and it is false. They feel counterintuitive
that way.

Perhaps
- new add on applies to all existing windows and workspaces and all newly
created workspaces
- user can deselect windows and workspaces
- if they deflect a workspace then it will not apply to new and existing
windows and the option disappears (or disabled)
